src/convert-workspace.ts

```diff
@@ -63,6 +63,19 @@ async function writePackageJson(
   writeJson(packageJson, packageJsonPath);
 }
 
+
+/**
+ * For a given repo, generate a new package.json and write it to disk.
+ */
+async function writeConversionManifest(
+    repo: WorkspaceRepo, converter: ProjectConverter) {
+  const bowerPackageName = path.basename(repo.dir);
+  const manifestJsonPath = path.join(repo.dir, 'manifest.json');
```

Comment:
'manifest.json' seems a bit generic; maybe this should include 'modulizer' in the name ('modulizer.manifest.json')?